### DISCUSSION TITLE
[Feature Request] reflect show mode via query parameter

### DISCUSSION THREAD

**Discussion by MH4GF** [liam-hq/liam#336] [Ideas]
Currently the default show mode is `Table name only`. However, there are cases where you may want to switch defaults.
Active and hidden tables can be reflect via query parameters. I would also like to reflect show mode from the query parameter.

**Comment by FunamaYukina** [comment]
We've implemented the ability to customize the default show mode via query parameters😄.
For example, using ``?showMode=ALL_FIELDS`` will display all fields for the table. This gives you more flexibility in tailoring your view.

**Reply by MH4GF** [reply]
ref: https://github.com/liam-hq/liam/pull/452
